### PR TITLE
fix: resolve issue #597

### PR DESCRIPTION
**Pull request by NaturezzZ**
The frontend only parses country name in Chinese. Without this PR, `location_country` is in local language (French in France, Deutsch in Germany), which causes the frontend unable to resolve country count.

With this PR, issue #597 is fixed.

### CODE REVIEW

**Comment by yihong0618** [comment]
@NaturezzZ nice catch!
@ben-29 this can be cherry-pick to workouts-page I think

**Review by yihong0618 — APPROVED** [review]

File: run_page/generator/db.py [hunk 1]
```diff
@@ -83,7 +83,9 @@ def update_or_create_activity(session, run_activity):
             if not location_country and start_point or location_country == "China":
                 try:
                     location_country = str(
-                        g.reverse(f"{start_point.lat}, {start_point.lon}")
+                        g.reverse(
+                            f"{start_point.lat}, {start_point.lon}", language="zh-CN"
```
Comment (ben-29) [inline run_page/generator/db.py:87]: you may also do this in the `except` block below

**Review by ben-29 — COMMENTED** [review]
Do we need a flag for this? What if someone from another country uses this project?

**Comment by yihong0618** [comment]
> Do we need a flag for this? What if someone from another country uses this project?

no need, we do not open the map easter for other language....

**Comment by NaturezzZ** [comment]
> > Do we need a flag for this? What if someone from another country uses this project?
> 
> no need, we do not open the map easter egg for other languages....

Yes, and only the visited provinces are highlighted. Currently we do not highlight countries other than China.

File: run_page/generator/db.py (hunk 1 above)
Comment (NaturezzZ) [reply run_page/generator/db.py:87]: done

**Review by ben-29 — APPROVED** [review]

**Pull request merged**